src/main.ts

```diff
@@ -75,6 +75,8 @@ export const verifyPassURIOffline = (
     return {
       success: true,
       violates: null,
+      exp: cwtClaims.exp ? cwtClaims.exp : null,
+      nbf: cwtClaims.nbf ? cwtClaims.nbf : null,
       credentialSubject,
```

Comment:
It'd be great if we can add a `claims: cwtClaims` property here with the full claims data

Reply:
In that case we should just returns `cwtClaims` imo, since `exp` and `nbf` are inside.